api/agent/drivers/docker/cookie.go

```diff
@@ -25,6 +26,56 @@ type cookie struct {
 	drv *DockerDriver
 }
 
+func (c *cookie) configureLogger(log logrus.FieldLogger) {
+
+	c.opts.HostConfig.LogConfig = docker.LogConfig{
+		Type: "none",
+	}
+
+	// Let's stream logs back to agent if docker syslog is disabled.
+	if !c.drv.conf.EnableDockerSyslog {
+		c.opts.Config.AttachStderr = true
+		return
+	}
+
+	conf := c.task.LoggerConfig()
+
+	// If docker-syslog is enabled and Task has no URL, we discard logs.
```

Comment:
this seems wrong, too... we still want to get stderr out right now for storing logs (locally) and eventually for debug mode we'll likely want the same? doesn't seem like the right place to be doing this, the creator of the cookie should pick this stuff